RELEASE.md

```diff
@@ -12,7 +12,25 @@
 # Upcoming Release 0.18.3
 
 ## Major features and improvements
-* Implemented autodiscovery of project pipelines.
+* Implemented autodiscovery of project pipelines. By default, the simplified `register_pipelines()` function in `src/<package_name>/pipeline_registry.py` looks like:
```

Comment:
I think we should make the point of this feature a bit clearer and state it upfront, since it's buried a bit after the function definition at the moment. Something along these lines:
```suggestion
* Implemented autodiscovery of project pipelines. A pipeline created with `kedro pipeline create <pipeline_name>` can now be made immediately available  in `kedro run --pipeline=<pipeline_name>` without needing to explicitly register it in `src/<package_name>/pipeline_registry.py`. By default, the simplified `register_pipelines()` function in `pipeline_registry.py` looks like:
```